spec/model/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  # tests go here
+  subject { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+  it 'PostsCounter must be an integer greater than or equal to zero' do
+    subject.posts_counter = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Good job working on these tests, but this does not validate that the `post_counter` is an integer. Please kindly include tests to actually validate that it is an integer and that it is not less than zero